docker/initdb.d/02-data.sql

```diff
@@ -222,6 +222,10 @@ INSERT INTO `mirror_aliases` (`id`, `alias`, `related_product`) VALUES (5,'firef
 INSERT INTO `mirror_aliases` (`id`, `alias`, `related_product`) VALUES (6,'firefox-devedition-latest','Devedition-128.0b1');
 INSERT INTO `mirror_aliases` (`id`, `alias`, `related_product`) VALUES (7,'firefox-latest-ssl','Firefox-SSL');
 INSERT INTO `mirror_aliases` (`id`, `alias`, `related_product`) VALUES (8,'partner-firefox-release-unitedinternet-foo-latest','Firefox-partner-unitedinternet-foo');
+INSERT INTO `mirror_aliases` (`id`, `alias`, `related_product`) VALUES (9,'firefox-beta-stub','Firefox-stub');
+INSERT INTO `mirror_aliases` (`id`, `alias`, `related_product`) VALUES (10,'firefox-devedition-stub','Firefox-stub');
+INSERT INTO `mirror_aliases` (`id`, `alias`, `related_product`) VALUES (11,'firefox-devedition-msi-latest-ssl','Firefox-beta-msi-latest-SSL');
+INSERT INTO `mirror_aliases` (`id`, `alias`, `related_product`) VALUES (12,'thunderbird-latest-ssl','Thunderbird-SSL');
```

Comment:
I guess these are to avoid duplicating all the products that your script might query?

Reply:
yes